docs/In-Band Manageability User Guide - INBS.md

```diff
@@ -45,3 +45,37 @@ To disable TLS (for example, for testing), set "tls_enabled" to false. tls_cert_
 - **tls_enabled**: Boolean value to enable or disable TLS. Set to `true` to enable TLS.
 - **tls_cert_path**: (ignored if TLS is disabled) The system path to the TLS certificate used for establishing a secure connection to the INBS server. It is recommended that the path be under `/etc/intel-manageability/public/` to ensure the certificate is accessible to the cloudadapter agent.
 - **token_path**: (ignored if TLS is disabled) Path to the file containing the confidential authentication token used to verify the device with the INBS server. It is recommended that the path be under `/etc/intel-manageability/secret/` to ensure the token is secure.
+
+### UDM Bootstrap Mode
```

Comment:
Will need to revisit the User Guide for INBS (UDM mode). But its a lower priority now. 

I am thinking we need to put instructions which assumes that the Admin has run the `mctl create bootstrap-token` cmd and has the docker command to be run on the EN.

We can have provide debug/dev info like the path or files it creates in UDM mode. (which you have added in this PR already)